frontend/pages/profile/MyProfile.tsx

```diff
@@ -1,48 +1,94 @@
-import * as React from 'react';
-import {useState, useEffect} from 'react';
-import {
-  Container,
-  Box,
-  Stack,
-  useBreakpointValue,
-} from '@chakra-ui/react';
+import * as React from "react";
+import { useState, useEffect } from "react";
+import { Container, Box, Stack, useBreakpointValue } from "@chakra-ui/react";
 
 import Navbar from "../../components/shared/Navbar";
 import Header from "../../components/profile/Header";
 import MyEpisodes from "../../components/profile/MyEpisodes";
 import Podcasts from "../../components/profile/Podcasts";
-import { useSession } from "next-auth/react";
-
-//For later to subscribe to podcasts {session?.user?.subscribedPodcastsID}
+import { UserProfile } from "../../utilities/Interfaces";
+//import { useSession } from "next-auth/react";
 //const { data: session } = useSession();
+//For later to subscribe to podcasts {session?.user?.subscribedPodcastsID}
+
+import { Router, useRouter } from "next/router";
+import UserProfileHelper from "../../helpers/UserProfileHelper";
+
 
 const myProfile = () => {
-const isInline = useBreakpointValue({ base: false, md: true, default: false });
+
+  const [userProfile, setUserProfile] = useState<UserProfile | undefined>(undefined);
```

Comment:
Why don't you just set the object instead of setting each field individually?

Reply:
you are right, it's redundant to set each field individually